app/src/main/java/com/android/sooz/filterapp/ThemeHeaderFragmentActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.android.sooz.filterapp;
+
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+
+public abstract class ThemeHeaderFragmentActivity extends AppCompatActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstance) {
+        super.onCreate(savedInstance);
+        setContentView(R.layout.activity_back_activity);
+
+        View container = findViewById(R.id.container);
```

Comment:
Just a minor cleanup thing: It doesn't look like this line is used and it can probably be deleted.